src/widgets/pictures/ui/pictures.scss

```diff
@@ -1,85 +1,46 @@
-.pictures {
-  & .social-media-container {
-    @include media-tablet {
-      flex-wrap: wrap;
-      align-items: center;
-      justify-content: center;
-    }
-
-    display: flex;
-    align-items: flex-start;
-  }
-}
-
-.pictures .carousel-root {
-  @include media-laptop {
-    width: 100%;
-    max-width: 730px;
-    height: 369px;
-  }
-
-  @include media-tablet {
-    width: 100%;
-    max-width: 358px;
-    height: 210px;
-    padding: 0 0 25px;
-  }
-
-  @include media-mobile {
-    width: 100%;
-    height: auto;
+.carousel {
+  .slide {
+    cursor: pointer;
+    padding: 0 5px;
   }
 
-  width: 1200px;
-  height: 540px;
-  margin: 16px auto;
-  padding: 10px 10px 40px;
-}
+  .carousel-slider {
+    .control-next,
+    .control-prev {
+      top: calc(50% - 50px);
 
-.carousel {
-  @include media-laptop {
-    height: 369px;
-  }
+      height: 100px;
 
-  @include media-tablet {
-    height: 210px;
+      opacity: $opacity-100;
+      backdrop-filter: blur(20px);
+      border-radius: 8px;
+    }
   }
 
-  height: 540px;
-}
+  .control-dots {
+    left: calc(50% - 150px);
```

Comment:
And here also need a variable because i have no idea what is this `150px`

Reply:
Done